### ISSUE TITLE
Create a PDF for hindi

### ISSUE THREAD

**Issue by starlightknown** [github/training-kit#835]
## Overview
**TL;DR**
Can I add a PDF for hindi version? I'd like to know how can I contribute to it if I can. Thank you

#### Screenshots
<If bug fix, please upload images of the error to better showcase the problem.>

#### Reasoning
<If new idea or direction, please create a checklist of proposed ideas or next steps. [Example](https://github.com/github/training-kit/issues/415).>

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it!
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://lab.github.com/githubtraining/introduction-to-github) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository.
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/`.

### Questions?
- Leave a comment on this issue! Make sure to use `@mentions` if you want a specific person's attention!

**Comment by YaLublu** [comment]
Super

**Comment by YaLublu** [comment]
Go

**Comment by hectorsector** [comment]
> Can I add a PDF for hindi version? I'd like to know how can I contribute to it if I can. Thank you

Of course @starlightknown! You can follow the instructions in [`CONTRIBUTING.md`](https://github.com/github/training-kit/blob/master/CONTRIBUTING.md) to translate a cheatsheet.

**Comment by kingworlds377** [comment]
All people hello how are you doing today are the beautiful people and tigers and lions are here for the world love love and love more of world
